packages/nextjs/app/_components/CommunityGrant.tsx

```diff
@@ -0,0 +1,82 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export const CommunityGrant = () => {
+  return (
+    <div id="communityGrants" className="bg-secondary">
+      <div className="container max-w-[90%] sm:max-w-md lg:max-w-7xl m-auto xl:pl-18 lg:pl-8 flex flex-col-reverse lg:flex-row gap-5 lg:gap-0 max-h-[30rem]">
```

Comment:
Maybe we could add some marging for smaller devices (those where we are showing the full orange shape), something like: `my-6 lg:my-0`

Reply:
Yup I think it makes sense, update at https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/25/commits/7cef5a5f30ffc65ce936aafbfdd5c0773a57c133, Thanks Pablo 🙌